File: Python4/NumeroPrimo.py
def es_primo(numero):
    #Descartamos que pueda ser divisible entre dos
    if numero % 2 == 0:
        return "No es primo" #Si es par
    else:
        return "Es primo" #Si es impar (primo)

#Introducir el numero
entrada = int(input("Introduce el número: "))
print(es_primo(entrada))

File: EJ1/NumeroPrimo.php
<?php

$numero = readline("Introduce el numero que quiera comprobar: ");

for ($i = 2; $i < $numero ;$i++) {
    if ($numero % $i == 0) {
        echo 'El número no es primo';
        break;
    } else {
        echo 'El número es primo';
        break;
    }
}

?>